### DISCUSSION TITLE
El buscador de las tasks, no está funcionando para aquellas tasks que se encuentran en un estado "Closed"

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#360] [4. Bug Reports] [answered]
Hola

Me lo levantó un cliente indicando que el buscador no funciona en aquellas tasks que se encuentran en un estado de tipo "**CLOSED**".

Lo repliqué y entrego los siguientes ss.

1.- En este caso quería buscar con el texto que indica y no me arrojó el resultado
![image](https://user-images.githubusercontent.com/61419471/127550620-73e09c4b-816a-4309-967d-900aa4ae9323.png)

2.- Este es otro ejemplo que ingreso el nombre completo de la tasks y me arroja otras tasks en estados de tipo "In-Progress"
![image](https://user-images.githubusercontent.com/61419471/127550968-314b2ac6-663c-4b56-bab6-14721b4a43cf.png)

Me podrían ayudar confirmando que solo se puede ver en estados de tipo **New** e **In-Progress**.

Gracias =>

**Comment by NicolasPLG** [comment, marked as answer]
Hola Probé en cambiar los estados "Closed" a "In-progress" desde la base y aún no se puede realzar las consultas :(

**Reply by NicolasPLG** [reply]
Durante la reunión nos dimos cuenta que el buscador está asociado al campo "**Ocultar tareas cerradas después de***". Se estaba usando 7 días y por eso no se encontraba en el buscador de task.

Está bien el comportamiento de "ese campo".

Se deja un ss.

![image](https://user-images.githubusercontent.com/61419471/127897504-e36d151f-d41c-446c-8628-33dfc7c16f75.png)

Ahora este cliente, como es temas de salud, los chequeos de pacientes pueden ser más de 1 año (actualmente lo dejamos en 10 años).

:(

**Reply by acedward** [reply]
Estamos trabajando en un nuevo release del buscador, debería estar disponible a principios del mes de Septiembre.
Si necesitas esto de forma urgente por favor ponte en contacto con tu representante para corregir la versión actual.

**Reply by NicolasPLG** [reply]
hola @acedward aa enteindo, si está en un nuevo release, no hay problema se espera a la fecha que estará =>

Muchas gracias
